.azure/pipelines/ci.yml

```diff
@@ -422,126 +427,128 @@ jobs:
-        inputs:
-          artifactName: artifacts-Windows-Installers
-          downloadPath: $(Build.StagingDirectory)/downloaded_artifacts/
-      - task: CopyFiles@2
-        displayName: Copy Windows artifacts to .deps/assets
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-Release/build/
-          contents: '**/*.tgz'
-          targetFolder: $(Build.SourcesDirectory)/.deps/assets/
-          flattenFolders: true
-      - task: CopyFiles@2
-        displayName: Copy Windows SharedFx artifacts to .deps/assets
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-SharedFx/Signed/
-          contents: |
-            SharedFx/**
-            OobArchives/**
-          targetFolder: $(Build.SourcesDirectory)/.deps/assets/
-          flattenFolders: true
-      - task: CopyFiles@2
-        displayName: Copy Package Archive artifacts to .deps/assets
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Package-Archive/lzma/
-          contents: '**'
-          targetFolder: $(Build.SourcesDirectory)/.deps/assets/
-      - task: CopyFiles@2
-        displayName: Copy SharedFx Installer artifacts to .deps/assets
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-SharedFx-Installers/installers/
-          contents: '**'
-          targetFolder: $(Build.SourcesDirectory)/.deps/assets/
-      - task: CopyFiles@2
-        displayName: Copy Windows Installer artifacts to .deps/assets
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-Installers/bin/Release/installers/
-          contents: |
-            en-US/*.msi
-            **/*.exe
-            **/*.wixlib
-            **/*.nupkg
-          targetFolder: $(Build.SourcesDirectory)/.deps/assets/
-          flattenFolders: true
-      - task: CopyFiles@2
-        displayName: Copy Windows SharedFx artifacts to .deps/packages
-        inputs:
-          sourceFolder: $(Build.StagingDirectory)/downloaded_artifacts/artifacts-Windows-SharedFx/Signed/Packages/
-          contents: '**'
-          targetFolder: $(Build.SourcesDirectory)/.deps/packages/
-      - task: DeleteFiles@1
-        displayName: Delete korebuild.json
-        inputs:
-          contents: korebuild.json
-      - script: .\build.cmd
-                -ci
-                $(BuildNumberArg)
-                /t:Publish
-                /p:BuildBranch=$(Build.SourceBranchName)
-                /bl:artifacts/logs/Publish.binlog
-        env:
-          PB_PACKAGEVERSIONPROPSURL: $(PB_PackageVersionPropsUrl)
-          PB_ASSETROOTURL: $(PB_AssetRootUrl)
-          PB_RESTORESOURCE: $(PB_RestoreSource)
-          PB_PUBLISHBLOBFEEDKEY: $(PB_PublishBlobFeedKey)
+    # Only run the publish job if this is a prodcon build
+    - ${{ if variables['ProductBuildId'] }}:
```

Comment:
Why disable the `Publish` target outside ProdCon❔ The uploaded artifact brings everything together nicely.

Reply:
Because the only assets we need are available as build output, this removes the need to do any blob publishing/maintain extra secrets

Reply:
So the build artifacts are already created and updated to the build info..this just skips the prodcon-style publishing unless running in prodcon?

Reply:
Yes